static/templates/actors/identify-item.hbs

```diff
@@ -21,7 +21,7 @@
                 </tr>
             </tbody>
         </table>
-    {{else if isAlchemical}}
+    {{else}}
         <h1>{{localize "PF2E.identification.IdentifyAlchemyDCs"}}</h1>
```

Comment:
I don't think we'll want to show "Identify Alchemy DCs" for non-alchemical items.

Reply:
Right, i confused myself with the part after that, the `{{localize "PF2E.Skill.Crafting"}}`, because both were using the same label there.